src/EnergyPlus/DataDaylighting.hh

```diff
@@ -63,27 +62,31 @@
 
 namespace EnergyPlus {
 
-namespace DataDaylighting {
+namespace Dayltg {
 
     // Two kinds of reference points: used directly in daylighting, used to show illuminance map of zone
     constexpr int MaxMapRefPoints(2500); // Maximum number of Illuminance Map Ref Points
 
-    enum class SkyType : int
+    enum class SkyType 
     {
         Invalid = -1,
-        Clear = 1,
+        Clear,
```

Comment:
`SkyType` now begins at `0` because it is used to index a `std::array` rather than Fortran arrays.